### DISCUSSION TITLE
Batch update rollback

### DISCUSSION THREAD

**Discussion by patvong** [directus/directus#2870] [Feature Requests]
# Feature Request
Ability to rollback a batch update.
Trying to revive this old feature request: https://github.com/directus/api/issues/123

## What problem does this feature solve?
A content manager user on Directus could perform a batch update with incorrect data. I would be nice to have a way to rollback easily from the UI.

## How do you think this should be implemented?
From the Activity Log UI, we can identify a batch update activity.
When loading the batch update activity detail, we can see a button to rollback the changes performed by the batch update.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
